src/ripple/app/paths/FindPaths.cpp

```diff
@@ -124,14 +134,15 @@ bool findPathsForOneIssuer (
         dstAccount,
         srcIssue.currency,
         srcIssue.account,
-        dstAmount);
+        dstAmount,
+        boost::none);
 
-    if (!pf.findPaths (searchLevel))
-        return false;
+    if (!pf.findPaths(searchLevel))
```

Comment:
Quibble:  might be better able to capture the RVO with:

    boost::optional<std::pair<STPathSet, std::vector<STAmount>>> result;
    if (pf.findPaths(searchLevel))
    {
        // compute path ranks and path.
    }
    return result;